internal/server/routes.go

```diff
@@ -15,33 +17,60 @@ import (
 	"github.com/go-chi/chi/v5/middleware"
 )
 
-func (s *Server) RegisterPatientRoutes(r *chi.Mux, patientRepository repositories.Repository[models.PatientInfo]) {
+func (s *Server) RegisterPatientRoutes(r *chi.Mux, patientRepository repositories.Repository[models.Patient]) {
 	patientHandler := patient.NewPatientHandler(patient.PatientHandlerConfig{
 		PatientRepo: patientRepository,
 		Cache:       s.cache,
 		Email:       s.email,
 	})
 
-	r.Get("/patient_signup", templ.Handler(web.PatientSignupPage()).ServeHTTP)
-	r.Post("/patient_submit", patientHandler.Signup)
-	r.Post("/patient_login", patientHandler.Login)
-	r.Get("/patient_dashboard", templ.Handler(web.PatientDashboardPage()).ServeHTTP)
-	r.Get("/appointments", patientHandler.Appointment)
-	r.Get("/generate_captcha",patientHandler.GenerateCaptcha)
```

Comment:
Please accomodate the generate_captcha and verify_captcha routes too to the changed logic